integration_tests/models/schema.yml

```diff
@@ -40,6 +40,13 @@ models:
           config:
             severity: warn
           tags: ["table_anomalies"]
+      - elementary.volume_anomalies:
+          time_bucket:
+            period: week
+            count: 1
+          config:
+            severity: warn
+          tags: ["table_anomalies"]
```

Comment:
Changing in here requires changing in the python E2E test - but I didn't find another PR in the python.

Reply:
Consulted with Or and decided not to add specific tests here at the moment

Reply:
What I meant here was - after adding this to the E2E we can expect that the report to look a bit different. We have a test in the E2E with very specific hard coded  expectations about what is going on in the package's E2E and it needs to be updated when the package's E2E changes. I ran this https://github.com/elementary-data/elementary/actions/runs/4524326381/jobs/7968070443 so I see there's no failures so no need to add anything.